### PR TITLE
Improving deepspeed.ai website

### PR DESCRIPTION
**Pull request by conglongli**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This is the most recommended way to read the documentation.
+
+## 2. Directly read files in this directory
+
+We do not recommend this way because this directory is organized to build the [deepspeed.ai](https://www.deepspeed.ai/) website using Jekyll. Thus some of the files actually are not DeepSpeed documentation. In addition, some of the url links in the documentation only work through the webpages generated by Jekyll.
+
+## 3. Build [deepspeed.ai](https://www.deepspeed.ai/) website locally using Jekyll
+
+This is recommended when you do not have internet access. You can follow the instruction at [here](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll) to install Ruby, Bundler, and Jekyll. Then run `bundle exec jekyll serve` at this directory so that you can view the website in you web browser at `http://localhost:4000`.
```
Comment (ShadenSmith) [inline docs/README.md:15]: typo "you web" -> "your web"

Comment (ShadenSmith) [inline docs/README.md:15]: Maybe also also add a mention to point 3 to say it's recommended for local website dev?

Comment (conglongli) [reply docs/README.md]: Fixed in the new commit.

Comment (conglongli) [reply docs/README.md]: Added in the new commit.

**Pull request merged**